content/release/1.3.0.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Release 1.3.0 available
+date: 2021-11-30
+linked: true
+aliases:
+    - /release/1.3.0
```

Comment:
Why is this alias needed?  Seems to be the same as the real name.

Reply:
Yeah this was used [previously](https://github.com/apache/ozone-site/commit/89cdb07b59cc6610f5d0402b846c750b6c8b28a2) to redirect 1.2.0 to 1.2.1. We shouldn't need it here for 1.3.0.